### PR TITLE
docs: packaging & docs

### PR DESCRIPTION
**Pull request by igorbernstein2**
Stacked PRs:
 * __->__#9695
 * #9692
 * #9691
 * #9690
 * #9689
 * #9688

--- --- ---

### docs: packaging & docs

Also:
* fix metric name

### CODE REVIEW

File: bigtable/bigtable-proxy/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+* `bigtableproxy.client.call.max_outstanding_count` Maximum number of concurrent RPCs in a single
+  minute window
+
+## Requirements
+
+* JVM >= 11
+* Ensure that the service account includes the IAM roles:
+  * `Monitoring Metric Writer`
+  * `Bigtable User`
+* Ensure that the metrics project has `Stackdriver Monitoring API` enabled
```
Comment (mutianf) [inline bigtable/bigtable-proxy/README.md:58]:
```suggestion
* Ensure that the Bigtable project has `Stackdriver Monitoring API` enabled
```

File: bigtable/bigtable-proxy/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# use the binary
+unzip target/bigtable-proxy-0.0.1-SNAPSHOT-bin.zip
+cd bigtable-proxy-0.0.1-SNAPSHOT
+./bigtable-proxy.sh \
+  --listen-port=1234 \
+  --metrics-project-id=SOME_GCP_PROJECT
+ 
+export BIGTABLE_EMULATOR_HOST=1234
+path/to/application/with/bigtable/client
```
Comment (mutianf) [inline bigtable/bigtable-proxy/README.md:74]: can these be one line?

export BIGTABLE_EMULATOR_HOST=1234 path/to/application/with/bigtable/client

File: bigtable/bigtable-proxy/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Required options:
+* `--listen-port=<port>` The local port to listen for Bigtable client connections. This needs to 
+  match port in the `BIGTABLE_EMULATOR_HOST="localhost:<port>` environment variable passed to your
+  application.
+* `--metrics-project-id=<projectid>` The Google Cloud project that should be used to collect metrics
+  emitted from the proxy.
+
+Optional configuration:
+* The environment variable `GOOGLE_APPLICATION_CREDENTIALS` can be used to use a non-default service
```
Comment (mutianf) [inline bigtable/bigtable-proxy/README.md:87]: how about data / admin endpoint and user agent? should we mention them here?

**Review by mutianf — APPROVED** [review]

File: bigtable/bigtable-proxy/README.md (hunk 2 above)
Comment (igorbernstein2) [reply bigtable/bigtable-proxy/README.md:74]: they could be, but I prefer to make it explicit

File: bigtable/bigtable-proxy/README.md (hunk 3 above)
Comment (igorbernstein2) [reply bigtable/bigtable-proxy/README.md:87]: dont really wanna advertise that

**Review by bhshkh — APPROVED** [review]

**Pull request merged**